pkg/pdapi/pdapi.go

```diff
@@ -565,24 +572,20 @@ func (c *pdClient) EndEvictLeader(storeID uint64) error {
 		klog.Errorf("call DELETE method: %s failed,statusCode: %v,error: %v", apiURL, res.StatusCode, err2)
 	}
 
-	// pd will return an error with the body contains "scheduler not found" if the scheduler is not found
-	// this is not the standard response.
-	// so these lines are just a workaround for now:
-	//   - make a new request to get all schedulers
-	//   - return nil if the scheduler is not found
-	//
-	// when PD returns standard json response, we should get rid of this verbose code.
+	return nil
```

Comment:
1. What's the detailed response from PD if the scheduler does not exist, the `err` in L560 is nil? is this case covered in L565-L567?
2. We should return error in L570 `else` block.